src/EnergyPlus/Cache.hh

```diff
@@ -90,6 +90,27 @@ namespace Cache {
 
     void writeCache(EnergyPlusData &state);
 
+#define USE_REINTERPRET_CAST
```

Comment:
@Myoldmopar feel free to review this commit if you have a minute since there have been a few options floated around? It's been noted [here](https://github.com/NREL/EnergyPlus/blob/develop/third_party/ObjexxFCL/src/ObjexxFCL/bit.hh#L258) that the reinterpret_cast usage is UB, but it's obviously been working OK since we've been using it for a long while now. @amirroth noted offline that the UB is due to mismatched sizes of the cast-to and cast-from types. In this case ```sizeof(double) == sizeof(unsigned long long)``` so things should be OK from that perspective.